spec/models/bib_spec.rb

```diff
@@ -62,29 +54,23 @@
 
   describe "#is_research?" do
     it "should declare partner record as research" do
-      test_bib = test_bibs[0]
-
-      expect(test_bib[:bib].is_research?).to eq(test_bib[:result])
+      expect(partner_record.is_research?).to eq(true)
     end
 
     it "should declare a bib with 0 items as research" do
-      test_bib = test_bibs[1]
-      expect(test_bib[:bib].is_research?).to eq(test_bib[:result])
+      expect(zero_item_bib.is_research?).to eq(true)
```

Comment:
I'm a little confused about how this test passed before, and what exactly it was testing. Was it not actually covering this case?

Reply:
The mock item had a partner location. The order of the checks was rearranged. Before, this would have followed the bib with no item flow. Since that change, it only fits the partner item check.